src/resources/multiple/Review.py

```diff
@@ -0,0 +1,65 @@
+from typing_extensions import ParamSpec
+from bson import ObjectId
+from db.driver import DBdriver
+from flask_restful import Resource, reqparse
+
+class MultipleReview(Resource):
+
+  def __init__(self) -> None:
+    self.db = DBdriver()
+    self.drink_dne = {
+      "data": {
+        "res": None,
+        "err": "Review with the _id DNE"
+      }
+    }, 404
+    self.parser = reqparse.RequestParser(bundle_errors = True)
+
+  def get(self) -> tuple[dict, int]:
+    self.parser.add_argument("_ids", type = str, action = "append")
+    self.parser.add_argument("sample", type = int)
```

Comment:
Not sure if we need sample functionality here. I added it to MultipleDrink because the front-end asked for it. If they say we need it for the reviews, we should keep it. Otherwise, we should probably comment it out for now.

Reply:
Frontend, said we're good! Ignore the other comment :)